src/Router.jsx

```diff
@@ -0,0 +1,84 @@
+import { BrowserRouter, Route, Routes, Navigate } from 'react-router-dom';
+import {
+    BaseScreen,
+    ChatPage,
+    EditProfilePage,
+    ErrorPage,
+    FeedScreenPage,
+    HomePage,
+    ProfilePage
+} from './pages';
+import { ProtectedRoute } from './components';
+
+export const profileID = 'jP-2I-E7'; // TODO: This is temporary
+
+export const Router = () => {
+    return (
+        <BrowserRouter>
+            <Routes>
+                <Route exact path='/' element={<HomePage />} />
+
+                <Route exact path='*' element={<Navigate to='/not-found' />} />
+
+                <Route exact path='/not-found' element={<ErrorPage />} />
+
+                <Route
+                    exact
+                    path='/site/*'
+                    element={
+                        <BaseScreen>
+                            <Routes>
+                                <Route
+                                    exact
+                                    index
+                                    element={
+                                        <ProtectedRoute>
+                                            <FeedScreenPage />
+                                        </ProtectedRoute>
+                                    }
+                                />
+
+                                <Route
+                                    exact
+                                    path='/profile/:id'
+                                    element={
+                                        <ProtectedRoute>
+                                            <ProfilePage id={profileID} />
```

Comment:
idk what for this idis xd

Reply:
I just moved the router part to separate component to clean App.jsx a little. I needed to wrap our app with AuthContext so this seemed like a way to go. Let me know if this can stay like I did this.

Reply:
just remove this ID if you are here already, generally in react the amount of files doesn't limitate us, we can have shit ton of files and it won't affect the bundle (even if we would add lazy loading)

So yeah, if you decided that it would be meaningfull to separate these components just for readability it's totally fine

In App.tsx we always put like setup functions and components, something which have to run on first app initialization so yeah, if we would add anything to the App.tsx, with this routing, component probably would be unreadable and too long

Reply:
Yup that's was my thought process. I did exactly that